content/articles/building-lyrics-search-app-with-vanilla-javascript/index.md

```diff
@@ -0,0 +1,453 @@
+
+##Building a Lyrics Search app with Vanilla Javascript
+
+![hero image](/building-lyrics-search-app-with-vanilla-javascript/hero.jpg)
+
+Not knowing the lyrics of a song is a problem most song-lovers encounter, especially when it's a rap song.
+
+In this article, we will be creating a platform where users can search for lyrics by entering the artist name or title of the song.
+
+We will be creating the search app using HTML5, CSS3, Vanilla JS, async-await with fetch method, OVH API, and ES6 features like arrow functions.
+
+###Prerequisites
+- A code editor (Visual studio code.)
+- Some basic knowledge of HTML, CSS, and JAVASCRIPT.
+- A laptop with an internet connection
+
+### Structuring the lyrics search app with HTML 5.
+ To begin, create a folder named `lyrics-app`, then you create three files in the folder: `lyrics.html`, `lyrics.css` and `lyrics.js`
+
+In the `lyrics.html` file, include the code below to create the structure for the lyrics search app.
+
+```html
+
+<!DOCTYPE html>
+<html lang=”en”>
+<head>
+   <meta charset=”UTF-8”>
+   <meta http-equiv=”X-UA_Compatible” content=”IE=edge”>
+   <meta name=”viewport” content=”width=device-width,initial-scale=1.0”>
+   <link rel=”stylesheet” type=”text/css” href=”lyrics.css”/>
+   <title>Lyrics Search App</title>
+</head>
+<body>
+   <div class=”container”>
+           <div class=”intro-text”>
+                 <h1>Learn your favourite</h1>
+                 <h2>song lyrics</h2>
+           </div>
+           
+           <div id=”lyrics-search”>
+                   <form action=”#” id=”searchMe”>
+                         <input type=”text” id=”lyricSearch” placeholder=”artist name or song title”/>
+                    </form>
+                     <div id=”search-result”></div>
+           </div>
+   </div>
+<script src=”lyrics.js”></script>
+</body>
+</html>
```

Comment:
this html is buggy
It seems that the quotes in the code changed once you copied the code into your markdown file.

Kindly go over the code and make sure it has the correct quotes

here's what I ended up with when I tried to correct it:
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <link rel="stylesheet" href="lyrics.css" />
    <title>Lyrics Search App</title>
  </head>
  <body>
    <div class="container">
      <div class="intro-text">
        <h1>Learn your favourite</h1>
        <h2>song lyrics</h2>
      </div>
      <div id="lyrics-search">
        <form action="#" id="searchMe">
          <input
            type="text"
            name=""
            id="lyricSearch"
            placeholder="artist name or song title"
          />
        </form>
        <div id="search-result"></div>
      </div>
    </div>
    <script src="lyrics.js"></script>
  </body>
</html>

```

Reply:
@adrianmurage , for me to make changes to those quotes,  I would have to open a new PR, but my code editor didn't return errors concerning the quotes, only if you could suggest changes and I commit them